packages/env-utils/package.json

```diff
@@ -12,7 +12,6 @@
     },
     "dependencies": {
         "expo-localization": "^14.1.1",
```

Comment:
you can remove this as well

Reply:
3048f25d35f3ce7be06b4c9d4afeb3e21bc5bc65